app/js/components/tabs/tabcomponents/savedComponent.jsx

```diff
@@ -26,13 +26,15 @@ class SavedComponent extends React.Component {
       definitionDeleteJobs: []
     };
     // cohort related methods
-    this.getLastSavedCohorts = this.getLastSavedCohorts.bind(this);
+    // TODO: To be added when it's possible to sort the results fetched
+    // this.getLastSavedCohorts = this.getLastSavedCohorts.bind(this);
     this.searchSavedCohorts = this.searchSavedCohorts.bind(this);
     this.deleteCohort = this.deleteCohort.bind(this);
     this.downloadCohort = this.downloadCohort.bind(this);
     this.viewCohort = this.viewCohort.bind(this);
     // definition related methods
-    this.getLastSavedDefinitions = this.getLastSavedDefinitions.bind(this);
+    // TODO: To be added when it's possible to sort the results fetched
+    // this.getLastSavedDefinitions = this.getLastSavedDefinitions.bind(this);
```

Comment:
It is not recommended to leave commented out code simply because it may be needed in future. I think it may even be on our https://wiki.openmrs.org/display/docs/Java+Conventions